### PR TITLE
add limbo, ebase, evoland2 and brotato

### PR DESCRIPTION
**Pull request by peterge1998**
## Description

Fixes #(issue)

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:x: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* App Name (`package.name`) - added from icon request form
* App Name (`package.name`)
* App Name (`package.name`)

### Icons updated
* App Name (`package.name`)
* App Name (`package.name`)

### Icons linked
* App Name (linked `package.name` to `@drawable/package`)
* App Name (linked `package.name` to `@drawable/package`)

### CODE REVIEW

File: svgs/evoland2.svg
Comment (Chefski) [inline svgs/evoland2.svg:1]: Some strokes in this have decimal points. Make sure they're whole numbers.

File: svgs/limbo.svg
Comment (Chefski) [inline svgs/limbo.svg:1]: The 6px stroke seems unnecessary as the icon has lots of white space. I think 10px or even 12px would be better.

**Review by Chefski — CHANGES_REQUESTED** [review]

File: svgs/evoland2.svg
Comment (peterge1998) [reply svgs/evoland2.svg:1]: done

File: svgs/limbo.svg
Comment (peterge1998) [reply svgs/limbo.svg:1]: done

**Pull request merged**